### PR TITLE
Chore: Introducing a Grafana HALL_OF_FAME.md

### PR DESCRIPTION
**Pull request by mckn**
**What this PR does / why we need it**:
Added Hugo to Grafana hall of fame <3

**Which issue(s) this PR fixes**:
None, I'm afraid 😢

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

File: GOVERNANCE.md [hunk 1]
```diff
@@ -106,6 +105,12 @@ The current team members are:
 - Will Browne ([Grafana Labs](https://grafana.com/))
 - Zoltán Bedi ([Grafana Labs](https://grafana.com/))
 
+### Hall of fame 🕺🏽🌟
+
+List of previous team members that have had a big impact on the company or the product and contributed during a long period of time.
+
+- Hugo Häggmark ([School of applied technology](https://salt.study))
+
```
Comment (RichiH) [inline GOVERNANCE.md:113]:
```suggestion
### Previous members:

- Hugo Häggmark ([School of applied technology](https://salt.study))

```

**Review by RichiH — CHANGES_REQUESTED** [review]
I would prefer to keep the structure of the governance files the same across all projects so they are easier to verify and soon to template.

That would mean listing all previous members, irrespective of status. If a distinct list of extra honors is introduced, I would prefer "special thanks" or similar.

In any case, I believe we should avoid using emojis in policy documents.

In summary: What about leaving governance neutral and as minimal as possible while introducing a `HALL_OF_FAME.md` or similar? The advantage would be that it's not limited to people after they leave. It can be a live doc of distinction.

**Comment by marcusoftnet** [comment]
Welcome to Salt, @hugohaggmark !

File: GOVERNANCE.md [hunk 2]
```diff
@@ -106,6 +105,12 @@ The current team members are:
 - Will Browne ([Grafana Labs](https://grafana.com/))
 - Zoltán Bedi ([Grafana Labs](https://grafana.com/))
 
+### Hall of fame 🕺🏽🌟
```
Comment (bergquist) [inline GOVERNANCE.md:108]:
```suggestion
### Emeritus Members🕺🏽🌟
```

Many other projects use Emeritus for this.

```
What Is an Emeritus Core Maintainer?
In academia, when well-regarded faculty move on to other interests, the school may give them the title emeritus professor. This title confers honor on the recipient. In this way, the professor remains associated with the department and school in which she or he taught. But an emeritus professor is no longer required to teach courses, fill research obligations, or serve on committees.
```

Comment (mckn) [reply GOVERNANCE.md:108]: I really like this suggestion!

**Comment by mckn** [comment]
So how should I proceed with this? Given @RichiH feedback?

File: GOVERNANCE.md (hunk 2 above)
Comment (dprokop) [reply GOVERNANCE.md:108]: Same, think this makes a lot of sense

**Comment by bergquist** [comment]
> I agree with @RichiH on In any case, I believe we should avoid using emojis in policy documents.

:+1:

**Review by bergquist — APPROVED** [review]

**Review by dprokop — APPROVED** [review]

**Review by jackw — APPROVED** [review]

**Review by RichiH — COMMENTED** [review]
I would still prefer to put *all* former members into governance, simply as "Previous members:" like in https://prometheus.io/governance/ -- and create a distinct file for extra honors.

This would keep governance neutral and factual, and allow a more thorough and celebratory documentation of significant contributions for past, and potentially for current, members.

All that being said, I don't want to block this PR, either.

**Review by daniellee — APPROVED** [review]
LGTM

We will miss Hugo!

**Review by RichiH — APPROVED** [review]
LGTM

**Pull request merged**